plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1688,13 +1688,21 @@ elementgui.fluid.page_visual_and_properties=Visual and properties
 elementgui.fluid.block_properties=Fluid block properties
 elementgui.fluid.bucket_properties=Fluid bucket properties
 elementgui.fluid.fluid_properties=Fluid properties
+elementgui.fluid.can_multiply=Can sources multiply?
+elementgui.fluid.flow_rate=Flow rate:
+elementgui.fluid.level_decrease=Level decrease per block:
+elementgui.fluid.slope_find_distance=Slope find distance:
+elementgui.fluid.spawn_particles=Spawn drip particles?
+elementgui.fluid.drip_particle=Drip particle:
 elementgui.fluid.when_added=When block added
 elementgui.fluid.when_entity_collides=When mob/player collides block
 elementgui.fluid.texture_still=Still texture
 elementgui.fluid.texture_flowing=Flowing
+elementgui.fluid.forge_properties=Forge fluid properties
```

Comment:
Maybe rename this to "Modded fluid properties", to cover potential other modding apis if they support any of these

Reply:
Done